backend/pkg/boards/errors.go

```diff
@@ -0,0 +1,45 @@
+package boards
+
+import (
+	"fmt"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+	"time"
+)
+
+type ErrNewClientFailed struct {
+	Addr      string
+	Timestamp time.Time
+	Inner     error
+}
+
+func (err ErrNewClientFailed) Error() {
+	fmt.Sprintf("Error creating new client at %s", err.Timestamp.Format(time.RFC3339))
+}
```

Comment:
the printing logic should be handled if necessary by the code reciving the error. Errors should have a `String` function, if not they cannot be returned as an `error` type. Change this method into `String` and then return or print the error once you get it.